test/core_tests.js

```diff
@@ -49,7 +50,7 @@
       expected = "rotateZ(0deg) rotateY(0deg) rotateX(0deg) translate3d(1000px, 0px, 0px)";
       assert.strictEqual( actual, expected, "canvas.style.transform initialized correctly" );
 
-      // Normalize result for IE 11 and Safari.
+      // Normalize result for IE 11 and Safari
```

Comment:
Can JSCS or JSHINT handle this? It might be more useful if we could automate this kind of stuff to reduce the chances it's spotted in unrelated PRs.